database/redis/tag.go

```diff
@@ -7,11 +7,34 @@ import (
 	"github.com/go-redis/redis/v8"
 )
 
+// SyncSystemTags removes old system tags and creates new.
+func (connector *DbConnector) SyncSystemTags(tags []string) error {
```

Comment:
```suggestion
func (connector *DbConnector) ReplaceSystemTags(tags []string) error {
```
Как будто по тому, что функция делает лучше подойдёт `Replace`